cadquery/occ_impl/shapes.py

```diff
@@ -314,8 +314,7 @@ def fix(obj: TopoDS_Shape) -> TopoDS_Shape:
 
 class Shape(object):
     """
-        Represents a shape in the system.
-        Wrappers the FreeCAD apiSh
+    Represents a shape in the system.Wraps TopoDS_Shape.
```

Comment:
I don't understand what `system.Wraps` means here.

Reply:
Oh, it's the end of a sentence without a space. I was reading it as a class.method.